commentsComponent/schema.sql

```diff
@@ -0,0 +1,20 @@
+DROP DATABASE IF EXISTS commentsList;
+CREATE DATABASE commentsList;
+USE commentsList;
+
+CREATE TABLE users (
+  id INT AUTO_INCREMENT,
+  username VARCHAR (25),
```

Comment:
Your users table is missing a few fields. You still need to include:

1. Avatar --I recommend storing the avatar in your table as a file path to the Amazon S3 location. 
2. Location
3.  Followers count
4. Pro-user status

These fields are essential to implementing your hover functionality for comments.